### PR TITLE
Dockerize health check tunnel

### PR DESCRIPTION
**Pull request by CCLDArjun**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -25,6 +25,15 @@ services:
...
     networks:
       - quasar-local
+  conn:
+    build:
+      context: .
+      dockerfile: ./tunnel/Dockerfile
+    volumes:
+      - /home/sce/Quasar/test_ssh_keys/test_keys.pub:/app/test_keys.pub
+      - /home/sce/Quasar/test_ssh_keys/test_keys:/app/test_keys
+      - /home/sce/Quasar/test_ssh_keys/known_hosts:/app/known_hosts
```
Comment (evanugarte) [inline docker-compose.yml]: could these live in `~/.ssh` instead?

File: docker-compose.yml [hunk 2]
```diff
@@ -25,6 +25,15 @@ services:
       - "/etc/cups/ppd/:/etc/cups/ppd"
     networks:
       - quasar-local
+  conn:
+    build:
+      context: .
+      dockerfile: ./tunnel/Dockerfile
+    volumes:
+      - /home/sce/.ssh/id_rsa.pub:/app/test_keys.pub
```
Comment (evanugarte) [inline docker-compose.yml]: ~/.ssh is better than the absolute path

File: tunnel/tun.sh [hunk 3]
```diff
@@ -17,5 +17,7 @@ open_ssh_tunnel () {
 
 }
 
+chmod 600 ${DOCKER_CONTAINER_SSH_KEYS}
```
Comment (evanugarte) [inline tunnel/tun.sh:20]: add a comment as to why we're doing this

Comment (CCLDArjun) [reply tunnel/tun.sh:20]: #52 adds an explanation

**Review by evanugarte — APPROVED** [review]

**Pull request merged**